### ISSUE TITLE
Incorrect cutting/pasteing behavior in Pinta 

### ISSUE THREAD

**Issue by temginson** [PintaProject/Pinta#951]
**Description**
When you cut something out in Pinta, everything goes with it, even if you only marked a certain part with the wand selection. Expected behavior is that only the selected part should be cut out.
**To Reproduce**
use the magic wand to mark a selected pattern and then paste it in then you se that evrything got 

**Additional Info**
If applicable, add a screenshot or attach an example file that demonstrates the issue.

https://github.com/user-attachments/assets/d1457b29-5422-4559-83f7-7fd767b6b88b

**Version**
operating system windows 10 and version of Pinta 2.1.2

**Comment by cameronwhite** [comment]
Yeah it seems like the cut action is grabbing all of the contents inside the selection boundary, like in this example. Using the Move Selected Pixels tool behaves as expected though

https://github.com/user-attachments/assets/04b8b475-cffb-45c7-8471-70aafe8cbbc6

**Comment by temginson** [comment]
so this is a bug right?

**Comment by cameronwhite** [comment]
Yes, I agree it's a bug
